ftdetect/pandoc.vim

```diff
@@ -0,0 +1,16 @@
+augroup pandoc_ftdetect
+    autocmd!
+augroup END
```

Comment:
In ftdetect, we do not need to do augroup.

:help ftdetect

>     Note that there is no "augroup" command, this has already been done

Reply:
Thanks for pointing this out, I was unaware it was being done automatically for ftdetect files.